scalafmt-tests/src/test/scala/org/scalafmt/util/StyleMapTest.scala

```diff
@@ -89,4 +90,19 @@ class StyleMapTest extends FunSuite {
     assert(newStyle2.newlines.implicitParamListModifierForce == overrideValue)
   }
 
+  test("StyleMap.numEntries: unsafeCallSite for literalArgumentLists") {
+    val code =
+      """object a {
+        |  println(1, 2, 3, 4, 5, 6)
+        |  println(1, 2, 3, 4, 5, 6)
+        |  // scalafmt: { binPack.unsafeCallSite = false }
+        |  println(1, 2, 3, 4, 5, 6)
+        |}
+      """.stripMargin.parse[Source].get
+    val fops1 = new FormatOps(code, ScalafmtConfig.default)
+    val fops2 = new FormatOps(code, ScalafmtConfig(binPack = BinPack.enabled))
+    assertEquals(fops1.styleMap.numEntries, 7)
```

Comment:
What would be the different styles here? I thought it would be at max 2 - the default and the changed one. Could you maybe just add a comment why those numbers?

Reply:
will do. btw, not sure if you review one commit at a time (i make them specially), perhaps you would have seen how these numbers change with every commit.

Reply:
I haven't looked at each commit, but it helps, thanks! Though I still wonder why we end up with those numbers.

Reply:
added detailed breakdown for each commit, ptal

Reply:
Thanks! That's super useful!